src/user-guide/writing-workflows/external-triggers.rst

```diff
@@ -1,21 +1,47 @@
 .. _Section External Triggers:
 
-External Triggers (Xtriggers)
-=============================
```

Comment:
This title pertains to the whole document which includes xtriggers, ext-triggers and workflow-state triggers.

Reply:
Yeah, not intentional.